api4/system.go

```diff
@@ -402,6 +403,27 @@ func getAnalytics(c *Context, w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func getLatestVersion(c *Context, w http.ResponseWriter, r *http.Request) {
+	if *c.App.Config().ExperimentalSettings.RestrictSystemAdmin {
+		c.Err = model.NewAppError("latestVersion", "api.restricted_system_admin", nil, "", http.StatusForbidden)
+		return
+	}
+
+	resp, err := c.App.GetLatestVersion("https://api.github.com/repos/mattermost/mattermost-server/releases/latest")
```

Comment:
Since this is Github-specific anyway, can we limit the parameter to just the org/repo path like this?
```suggestion
	resp, err := c.App.GetLatestVersion("mattermost/mattermost-server")
```

Reply:
I wanted to pass in the whole URL so I can mock the GET request in my tests, I'll take a look and see if there's a way for me to incorporate this

Reply:
I'm 0/5 on this. I was thinking it might make the method easier to invoke, but if as a result it's less testable then it's probably not worth it.